protocols/v2/binary-sv2/serde-sv2/src/primitives/sequences/seq0255.rs

```diff
@@ -11,7 +11,9 @@ pub struct Seq0255<'s, T: Serialize + TryFromBSlice<'s> + Clone> {
     seq: Option<Seq<'s, T>>,
     data: Option<Vec<T>>,
 }
+// impl<'s> Eq for Seq064K<'s, B016M<'s>> {}
```

Comment:
There is no need to change this file. This file should remain exactly equal to the one that is the main branch.

Reply:
I added this in to resolve this error I was getting:
```
error[E0277]: the trait bound `Seq0255<'decoder, U256<'decoder>>: core::cmp::Eq` is not satisfied
   --> protocols/v2/subprotocols/template-distribution/src/new_template.rs:56:5
    |
56  |     pub merkle_path: Seq0255<'decoder, U256<'decoder>>,
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `core::cmp::Eq` is not implemented for `Seq0255<'decoder, U256<'decoder>>`
```
Is there an alternative to fix this error?

Reply:
I tried to remove it and I wasn't  getting errors. But if you get that error leave it there.